test/algorithms/multi_unique.cpp

```diff
@@ -8,18 +8,10 @@
 #include <geometry_test_common.hpp>
 
 #include <boost/geometry/algorithms/unique.hpp>
-#include <boost/geometry/multi/algorithms/unique.hpp>
-
-#include <boost/geometry/io/wkt/wkt.hpp>
```

Comment:
`io/wkt/wkt.hpp` is included in `test_unique.hpp`.
In that respect it is okay to remove from here.